legacy/messaging/src/main/java/org/jboss/as/messaging/MessagingTransformers.java

```diff
@@ -116,10 +87,21 @@ static void registerTransformers(final SubsystemRegistration subsystem) {
      */
     private static void buildTransformers2_1_0(ResourceTransformationDescriptionBuilder builder) {
         ResourceTransformationDescriptionBuilder hornetqServer = builder.addChildResource(pathElement(HORNETQ_SERVER));
-        rejectDefinedAttributeWithDefaultValue(hornetqServer, OVERRIDE_IN_VM_SECURITY);
-
         ResourceTransformationDescriptionBuilder addressSetting = hornetqServer.addChildResource(AddressSettingDefinition.PATH);
-        rejectDefinedAttributeWithDefaultValue(addressSetting, MAX_REDELIVERY_DELAY, REDELIVERY_MULTIPLIER, SLOW_CONSUMER_CHECK_PERIOD, SLOW_CONSUMER_POLICY, SLOW_CONSUMER_THRESHOLD);
+        rejectDefinedAttributeWithDefaultValue(addressSetting, MAX_REDELIVERY_DELAY, REDELIVERY_MULTIPLIER);
```

Comment:
`override-invm-security` and `slow-consumer-*`attributes must be handled because of an accidental change in messaging 1.4 XSD after it was release: https://issues.jboss.org/browse/WFLY-4533